src/modules/fancyzones/lib/FancyZones.cpp

```diff
@@ -84,9 +101,12 @@ struct FancyZones : public winrt::implements<FancyZones, IFancyZones, IFancyZone
     winrt::com_ptr<IZoneWindow> m_zoneWindowMoveSize; // "Active" ZoneWindow, where the move/size is happening. Will update as drag moves between monitors.
     IFancyZonesSettings* m_settings{};
     GUID m_currentVirtualDesktopId{}; // UUID of the current virtual desktop. Is GUID_NULL until first VD switch per session.
+    std::unordered_map<GUID, bool> m_virtualDesktopIds;
     wil::unique_handle m_terminateEditorEvent; // Handle of FancyZonesEditor.exe we launch and wait on
+    wil::unique_handle m_fancyZonesDestroyedEvent;

```

Comment:
Maybe we can call this event something more indicative of the fact that is used to quit the `virtualDesktopTrackerThread`.
For example `m_quitVirtualDesktopTrackerThread`